From 46e60905203165323a5d945d9580b15387445eb3 Mon Sep 17 00:00:00 2001
From: Robert Schweikert <rjschwei@suse.com>
Date: Wed, 7 May 2025 18:40:21 +0200
Subject: [PATCH 3/3] On OpenSUSE do not add to kernel command line

Networks arguments on the kernel command line are set during image build

there is no need for another place for a hard coded list.
---
 dracut/30afterburn/module-setup.sh | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/dracut/30afterburn/module-setup.sh b/dracut/30afterburn/module-setup.sh
index 17c5880..de70f1b 100755
--- a/dracut/30afterburn/module-setup.sh
+++ b/dracut/30afterburn/module-setup.sh
@@ -16,12 +16,8 @@ install() {
     inst_simple "$moddir/afterburn-hostname.service" \
         "$systemdutildir/system/afterburn-hostname.service"
 
-    inst_simple "$moddir/afterburn-network-kargs.service" \
-        "$systemdutildir/system/afterburn-network-kargs.service"
-
     # These services are only run once on first-boot, so they piggyback
     # on Ignition completion target.
     mkdir -p "$initdir/$systemdsystemunitdir/ignition-complete.target.requires"
     ln -s "../afterburn-hostname.service" "$initdir/$systemdsystemunitdir/ignition-complete.target.requires/afterburn-hostname.service"
-    ln -s "../afterburn-network-kargs.service" "$initdir/$systemdsystemunitdir/ignition-complete.target.requires/afterburn-network-kargs.service"
 }
-- 
2.47.2

